libs/MeshKernel/include/MeshKernel/BilinearInterpolationOnGriddedSamples.hpp

```diff
@@ -30,10 +30,19 @@
 #include <MeshKernel/Mesh2D.hpp>
 #include <MeshKernel/MeshInterpolation.hpp>
 
+#include <concepts>
+#include <span>
+
 namespace meshkernel
 {
+
+    template <typename T>
+    concept InterpolatableType =
+        std::same_as<T, short> ||
+        std::same_as<T, float>;
+
```

Comment:
Should double also be added to the list of InterpolatableTypes?
I guess, most of the time it will be double, only sometimes will it be something else